dev/scripts/src/alluxio.org/build-distribution/cmd/root.go

```diff
@@ -18,8 +18,9 @@ import (
 )
 
 var (
-	debugFlag bool
-	ufsModulesFlag string
+	debugFlag           bool
```

Comment:
reorder alphabetically; put `customUfsModuleFlag` at the top

Reply:
OK